devices/consts.go

```diff
@@ -0,0 +1,96 @@
+package devices
+
+import (
+	"fmt"
+	"regexp"
+)
+
+const (
+	DriverUioPciGeneric = "uio_pci_generic"
+	DriverIgUio         = "ig_uio"
+	DriverVfioPci       = "vfio-pci"
+	DriverVfioPciKoName = "vfio_pci"
+	DriverUioHvGeneric  = "uio_hv_generic"
+)
+
+const (
+	PathSysPciDevices     = "/sys/bus/pci/devices"
+	PathSysPciDrivers     = "/sys/bus/pci/drivers"
+	PathSysPciDriverProbe = "/sys/bus/pci/drivers_probe"
+)
+
+const (
+	PathSysVmbusDevices = "/sys/bus/vmbus/devices"
+	PathSysVmbusDrivers = "/sys/bus/vmbus/drivers"
+)
+
+type StringBuilder string
```

Comment:
Why do you need a new type if it is used only in this file, not exported and do not have new methods?